Move `etc` under detection_rules

Mikaayenson opened the pull request:
<!--
Thank you for your interest in and contributing to Detection Rules!
There are a few simple things to check before submitting your pull request
that can help with the review process. You should delete these items
from your submission, but they are here to help bring them to your attention.
-->

## Issues
<!-- Link to related issues. Use closing keywords when appropriate -->
Related to #1765 .

## Summary
Part 1 out 2: Minimalistic refactor to move etc directory under detection_rules directory and update paths since it involves so many files.

Mikaayenson commented:
@brokensound77 I can resolve the version.lock.json locally but we should take about how this will affect ci upstream.

brokensound77 commented on .github/CODEOWNERS:
```diff
@@ -6,8 +6,8 @@ detection_rules/  @brokensound77 @rw-access
-etc/*.json        @brokensound77 @rw-access
-etc/*.json        @brokensound77 @rw-access
-etc/*/*           @brokensound77 @rw-access
+detection_rules/etc/packages.yml  @brokensound77 @rw-access
```
drop `rw_access` and add yourself and @terrancedejesus

brokensound77 commented on detection_rules/rule_loader.py:
```diff
@@ -98,8 +98,8 @@ def load_locks_from_tag(remote: str, tag: str) -> (str, dict, dict):
     commit_hash = git('rev-list', '-1', tag)
-    version = json.loads(git('show', f'{tag}:etc/version.lock.json'))
-    deprecated = json.loads(git('show', f'{tag}:etc/deprecated_rules.json'))
+    version = json.loads(git('show', f'{tag}:detection_rules/etc/version.lock.json'))
```
anything referencing tagged data will need resiliency to account for the old directory structure

brokensound77 commented:
>@brokensound77 I can resolve the version.lock.json locally but we should take about how this will affect ci upstream and all the other moving pieces within other related branches.

I would let #1884 merge first before resolving the move

As far as upstream CI, I think we should be ok - I don't believe anything references the repo that would be impacted.

Where this can cause problems is with any code that makes references to tagged branches or any other frozen state of the repo

terrancedejesus commented:
LGTM. Saw some docstrings in `utils.py` that could change from `etc/` to `detection_rules/etc/` (lines - 93,98,104,110,115) so nothing major regarding functionality. Once the adjustments to `CODEOWNERS` is done, this looks good to go for part 1.

brokensound77 reviewed (approved):
If the following look good, then I think this is good to go 🎉 
* `make release` and compare hash to main
* run a sampling of CLI commands
* any other testing we think may apply

Thanks 👍

Mikaayenson commented:
# make release 

<details>

<summary>main</summary>

```

(detection_dev) ➜  detection-rules git:(1765-fr-move-to-setuppy-installation) git checkout main
Switched to branch 'main'
Your branch is up to date with 'origin/main'.
(detection_dev) ➜  detection-rules git:(main) make release
pip install virtualenv
Collecting virtualenv
  Downloading virtualenv-20.14.1-py2.py3-none-any.whl (8.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8.8/8.8 MB 19.0 MB/s eta 0:00:00
Collecting platformdirs<3,>=2
  Downloading platformdirs-2.5.2-py3-none-any.whl (14 kB)
Collecting distlib<1,>=0.3.1
  Using cached distlib-0.3.4-py2.py3-none-any.whl (461 kB)
Collecting filelock<4,>=3.2
  Using cached filelock-3.6.0-py3-none-any.whl (10.0 kB)
Requirement already satisfied: six<2,>=1.9.0 in /Users/stryker/.virtualenvs/detection_dev/lib/python3.10/site-packages (from virtualenv) (1.16.0)
Installing collected packages: distlib, platformdirs, filelock, virtualenv
Successfully installed distlib-0.3.4 filelock-3.6.0 platformdirs-2.5.2 virtualenv-20.14.1
virtualenv ./env/detection-rules-build --python=python3.8
created virtual environment CPython3.8.9.final.0-64 in 843ms
  creator CPython3macOsFramework(dest=/Users/stryker/workspace/Elastic/detection-rules/env/detection-rules-build, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/Users/stryker/Library/Application Support/virtualenv)
    added seed packages: pip==22.0.4, setuptools==61.2.0, wheel==0.37.1
  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
./env/detection-rules-build/bin/python -m pip install -r requirements.txt
Collecting jsl==0.2.4
  Using cached jsl-0.2.4.tar.gz (21 kB)
  Preparing metadata (setup.py) ... done
Collecting pytoml
  Using cached pytoml-0.1.21-py2.py3-none-any.whl (8.5 kB)
Collecting toml==0.10.0
  Using cached toml-0.10.0-py2.py3-none-any.whl (25 kB)
Collecting requests~=2.27
  Using cached requests-2.27.1-py2.py3-none-any.whl (63 kB)
Collecting Click==7.0
  Using cached Click-7.0-py2.py3-none-any.whl (81 kB)
Collecting PyYAML~=5.3
  Using cached PyYAML-5.4.1.tar.gz (175 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting eql==0.9.12
  Using cached eql-0.9.12-py2.py3-none-any.whl (105 kB)
Collecting elasticsearch~=8.1
  Using cached elasticsearch-8.1.3-py3-none-any.whl (373 kB)
Collecting XlsxWriter~=1.3.6
  Using cached XlsxWriter-1.3.9-py2.py3-none-any.whl (145 kB)
Collecting marshmallow~=3.13.0
  Using cached marshmallow-3.13.0-py2.py3-none-any.whl (47 kB)
Collecting marshmallow-dataclass[union]~=8.5.6
  Downloading marshmallow_dataclass-8.5.8-py3-none-any.whl (17 kB)
Collecting pyflakes==2.2.0
  Using cached pyflakes-2.2.0-py2.py3-none-any.whl (66 kB)
Collecting flake8==3.8.1
  Using cached flake8-3.8.1-py2.py3-none-any.whl (72 kB)
Collecting pep8-naming==0.7.0
  Using cached pep8_naming-0.7.0-py2.py3-none-any.whl (7.7 kB)
Collecting pytest>=3.6
  Downloading pytest-7.1.2-py3-none-any.whl (297 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 297.0/297.0 KB 4.4 MB/s eta 0:00:00
Collecting jsonschema==3.2.0
  Using cached jsonschema-3.2.0-py2.py3-none-any.whl (56 kB)
Collecting marshmallow-jsonschema~=0.12.0
  Using cached marshmallow_jsonschema-0.12.0-py3-none-any.whl (11 kB)
Collecting marshmallow-union~=0.1.15
  Using cached marshmallow_union-0.1.15.post1-py2.py3-none-any.whl (4.6 kB)
Collecting lark-parser~=0.11.1
  Using cached lark-parser-0.11.3.tar.gz (229 kB)
  Preparing metadata (setup.py) ... done
Collecting mccabe<0.7.0,>=0.6.0
  Using cached mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
Collecting pycodestyle<2.7.0,>=2.6.0a1
  Using cached pycodestyle-2.6.0-py2.py3-none-any.whl (41 kB)
Collecting flake8-polyfill<2,>=1.0.2
  Using cached flake8_polyfill-1.0.2-py2.py3-none-any.whl (7.3 kB)
Collecting attrs>=17.4.0
  Using cached attrs-21.4.0-py2.py3-none-any.whl (60 kB)
Collecting six>=1.11.0
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting pyrsistent>=0.14.0
  Downloading pyrsistent-0.18.1-cp38-cp38-macosx_10_9_universal2.whl (81 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 81.4/81.4 KB 2.7 MB/s eta 0:00:00
Requirement already satisfied: setuptools in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (61.2.0)
Collecting certifi>=2017.4.17
  Using cached certifi-2021.10.8-py2.py3-none-any.whl (149 kB)
Collecting charset-normalizer~=2.0.0
  Using cached charset_normalizer-2.0.12-py3-none-any.whl (39 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached urllib3-1.26.9-py2.py3-none-any.whl (138 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.3-py3-none-any.whl (61 kB)
Collecting elastic-transport<9,>=8
  Using cached elastic_transport-8.1.2-py3-none-any.whl (59 kB)
Collecting typing-inspect>=0.7.1
  Using cached typing_inspect-0.7.1-py3-none-any.whl (8.4 kB)
Collecting typeguard
  Using cached typeguard-2.13.3-py3-none-any.whl (17 kB)
Collecting iniconfig
  Using cached iniconfig-1.1.1-py2.py3-none-any.whl (5.0 kB)
Collecting pluggy<2.0,>=0.12
  Using cached pluggy-1.0.0-py2.py3-none-any.whl (13 kB)
Collecting py>=1.8.2
  Using cached py-1.11.0-py2.py3-none-any.whl (98 kB)
Collecting packaging
  Using cached packaging-21.3-py3-none-any.whl (40 kB)
Collecting tomli>=1.0.0
  Using cached tomli-2.0.1-py3-none-any.whl (12 kB)
Collecting mypy-extensions>=0.3.0
  Using cached mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
Collecting typing-extensions>=3.7.4
  Downloading typing_extensions-4.2.0-py3-none-any.whl (24 kB)
Collecting pyparsing!=3.0.5,>=2.0.2
  Downloading pyparsing-3.0.8-py3-none-any.whl (98 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 98.5/98.5 KB 3.8 MB/s eta 0:00:00
Building wheels for collected packages: jsl, PyYAML, lark-parser
  Building wheel for jsl (setup.py) ... done
  Created wheel for jsl: filename=jsl-0.2.4-py3-none-any.whl size=26240 sha256=9da0c9298e2c9bf43583654b7fc251689a8b2ba926fad13661bb2a1b81f3ba45
  Stored in directory: /Users/stryker/Library/Caches/pip/wheels/41/b7/98/ae8fab1e326486621662184e6543dcc4cc5917577b782abfae
  Building wheel for PyYAML (pyproject.toml) ... done
  Created wheel for PyYAML: filename=PyYAML-5.4.1-cp38-cp38-macosx_10_14_arm64.whl size=45661 sha256=e5a00c52f1e59d7a497bd1195c243ccfacce7198f19f9f70f86486ab5d3d10ab
  Stored in directory: /Users/stryker/Library/Caches/pip/wheels/dd/c5/1d/5d7436173d3efd4a14dcb510eb0b29525ecb6b0e41489e716e
  Building wheel for lark-parser (setup.py) ... done
  Created wheel for lark-parser: filename=lark_parser-0.11.3-py2.py3-none-any.whl size=99736 sha256=560d0518e7ab777b2a8085197a73f3116592940ce11dbbdcab7f56d93843f082
  Stored in directory: /Users/stryker/Library/Caches/pip/wheels/ab/4c/ed/d2b019c1776b4387fc0fe4a48b1e5cf31d3e2ec4c51c955af5
Successfully built jsl PyYAML lark-parser
Installing collected packages: XlsxWriter, toml, pytoml, mypy-extensions, mccabe, lark-parser, jsl, iniconfig, certifi, urllib3, typing-extensions, typeguard, tomli, six, PyYAML, pyrsistent, pyparsing, pyflakes, pycodestyle, py, pluggy, marshmallow, idna, eql, Click, charset-normalizer, attrs, typing-inspect, requests, packaging, marshmallow-union, marshmallow-jsonschema, jsonschema, flake8, elastic-transport, pytest, marshmallow-dataclass, flake8-polyfill, elasticsearch, pep8-naming
Successfully installed Click-7.0 PyYAML-5.4.1 XlsxWriter-1.3.9 attrs-21.4.0 certifi-2021.10.8 charset-normalizer-2.0.12 elastic-transport-8.1.2 elasticsearch-8.1.3 eql-0.9.12 flake8-3.8.1 flake8-polyfill-1.0.2 idna-3.3 iniconfig-1.1.1 jsl-0.2.4 jsonschema-3.2.0 lark-parser-0.11.3 marshmallow-3.13.0 marshmallow-dataclass-8.5.8 marshmallow-jsonschema-0.12.0 marshmallow-union-0.1.15.post1 mccabe-0.6.1 mypy-extensions-0.4.3 packaging-21.3 pep8-naming-0.7.0 pluggy-1.0.0 py-1.11.0 pycodestyle-2.6.0 pyflakes-2.2.0 pyparsing-3.0.8 pyrsistent-0.18.1 pytest-7.1.2 pytoml-0.1.21 requests-2.27.1 six-1.16.0 toml-0.10.0 tomli-2.0.1 typeguard-2.13.3 typing-extensions-4.2.0 typing-inspect-0.7.1 urllib3-1.26.9
./env/detection-rules-build/bin/python -m pip install setuptools -U
Requirement already satisfied: setuptools in ./env/detection-rules-build/lib/python3.8/site-packages (61.2.0)
Collecting setuptools
  Using cached setuptools-62.1.0-py3-none-any.whl (1.1 MB)
Installing collected packages: setuptools
  Attempting uninstall: setuptools
    Found existing installation: setuptools 61.2.0
    Uninstalling setuptools-61.2.0:
      Successfully uninstalled setuptools-61.2.0
Successfully installed setuptools-62.1.0
./env/detection-rules-build/bin/python -m pip install -r requirements.txt
Requirement already satisfied: jsl==0.2.4 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 1)) (0.2.4)
Requirement already satisfied: pytoml in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 2)) (0.1.21)
Requirement already satisfied: toml==0.10.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 3)) (0.10.0)
Requirement already satisfied: requests~=2.27 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 4)) (2.27.1)
Requirement already satisfied: Click==7.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 5)) (7.0)
Requirement already satisfied: PyYAML~=5.3 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 6)) (5.4.1)
Requirement already satisfied: eql==0.9.12 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 7)) (0.9.12)
Requirement already satisfied: elasticsearch~=8.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 8)) (8.1.3)
Requirement already satisfied: XlsxWriter~=1.3.6 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 9)) (1.3.9)
Requirement already satisfied: marshmallow~=3.13.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 10)) (3.13.0)
Requirement already satisfied: marshmallow-dataclass[union]~=8.5.6 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 11)) (8.5.8)
Requirement already satisfied: pyflakes==2.2.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 14)) (2.2.0)
Requirement already satisfied: flake8==3.8.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 15)) (3.8.1)
Requirement already satisfied: pep8-naming==0.7.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 16)) (0.7.0)
Requirement already satisfied: pytest>=3.6 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 17)) (7.1.2)
Requirement already satisfied: jsonschema==3.2.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 18)) (3.2.0)
Requirement already satisfied: marshmallow-jsonschema~=0.12.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 19)) (0.12.0)
Requirement already satisfied: marshmallow-union~=0.1.15 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 20)) (0.1.15.post1)
Requirement already satisfied: lark-parser~=0.11.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from eql==0.9.12->-r requirements.txt (line 7)) (0.11.3)
Requirement already satisfied: pycodestyle<2.7.0,>=2.6.0a1 in ./env/detection-rules-build/lib/python3.8/site-packages (from flake8==3.8.1->-r requirements.txt (line 15)) (2.6.0)
Requirement already satisfied: mccabe<0.7.0,>=0.6.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from flake8==3.8.1->-r requirements.txt (line 15)) (0.6.1)
Requirement already satisfied: flake8-polyfill<2,>=1.0.2 in ./env/detection-rules-build/lib/python3.8/site-packages (from pep8-naming==0.7.0->-r requirements.txt (line 16)) (1.0.2)
Requirement already satisfied: attrs>=17.4.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (21.4.0)
Requirement already satisfied: setuptools in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (62.1.0)
Requirement already satisfied: pyrsistent>=0.14.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (0.18.1)
Requirement already satisfied: six>=1.11.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (1.16.0)
Requirement already satisfied: idna<4,>=2.5 in ./env/detection-rules-build/lib/python3.8/site-packages (from requests~=2.27->-r requirements.txt (line 4)) (3.3)
Requirement already satisfied: charset-normalizer~=2.0.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from requests~=2.27->-r requirements.txt (line 4)) (2.0.12)
Requirement already satisfied: certifi>=2017.4.17 in ./env/detection-rules-build/lib/python3.8/site-packages (from requests~=2.27->-r requirements.txt (line 4)) (2021.10.8)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from requests~=2.27->-r requirements.txt (line 4)) (1.26.9)
Requirement already satisfied: elastic-transport<9,>=8 in ./env/detection-rules-build/lib/python3.8/site-packages (from elasticsearch~=8.1->-r requirements.txt (line 8)) (8.1.2)
Requirement already satisfied: typing-inspect>=0.7.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from marshmallow-dataclass[union]~=8.5.6->-r requirements.txt (line 11)) (0.7.1)
Requirement already satisfied: typeguard in ./env/detection-rules-build/lib/python3.8/site-packages (from marshmallow-dataclass[union]~=8.5.6->-r requirements.txt (line 11)) (2.13.3)
Requirement already satisfied: py>=1.8.2 in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (1.11.0)
Requirement already satisfied: packaging in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (21.3)
Requirement already satisfied: tomli>=1.0.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (2.0.1)
Requirement already satisfied: iniconfig in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (1.1.1)
Requirement already satisfied: pluggy<2.0,>=0.12 in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (1.0.0)
Requirement already satisfied: mypy-extensions>=0.3.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from typing-inspect>=0.7.1->marshmallow-dataclass[union]~=8.5.6->-r requirements.txt (line 11)) (0.4.3)
Requirement already satisfied: typing-extensions>=3.7.4 in ./env/detection-rules-build/lib/python3.8/site-packages (from typing-inspect>=0.7.1->marshmallow-dataclass[union]~=8.5.6->-r requirements.txt (line 11)) (4.2.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in ./env/detection-rules-build/lib/python3.8/site-packages (from packaging->pytest>=3.6->-r requirements.txt (line 17)) (3.0.8)
RELEASE:
./env/detection-rules-build/bin/python -m detection_rules dev build-release --generate-navigator

█▀▀▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄   ▄      █▀▀▄ ▄  ▄ ▄   ▄▄▄ ▄▄▄
█  █ █▄▄  █  █▄▄ █    █   █  █ █ █▀▄ █      █▄▄▀ █  █ █   █▄▄ █▄▄
█▄▄▀ █▄▄  █  █▄▄ █▄▄  █  ▄█▄ █▄█ █ ▀▄█      █ ▀▄ █▄▄█ █▄▄ █▄▄ ▄▄█

[+] Building package 8.3
 - 12 rules excluded from package
Rule changes detected!
 - 17 changed rules
 - 4 new rules
 - 0 newly deprecated rules
run `build-release --update-version-lock` to update version.lock.json and deprecated_rules.json
Package saved to: /Users/stryker/workspace/Elastic/detection-rules/releases/8.3
- sha256: 15aaf0fd89754cd0497fa30d35d4954a3c64d6d94d354beae10cedaa06ec9686
- 679 rules included
rm -rf dist
mkdir dist
cp -r releases/*/*.zip dist/
(detection_dev) ➜  detection-rules git:(main)

```

</details>


<details>

<summary>1765-fr-move-to-setuppy-installation</summary>

```

(detection_dev) ➜  detection-rules git:(1765-fr-move-to-setuppy-installation) make release
./env/detection-rules-build/bin/python -m pip install -r requirements.txt
Requirement already satisfied: jsl==0.2.4 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 1)) (0.2.4)
Requirement already satisfied: pytoml in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 2)) (0.1.21)
Requirement already satisfied: toml==0.10.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 3)) (0.10.0)
Requirement already satisfied: requests~=2.27 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 4)) (2.27.1)
Requirement already satisfied: Click==7.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 5)) (7.0)
Requirement already satisfied: PyYAML~=5.3 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 6)) (5.4.1)
Requirement already satisfied: eql==0.9.12 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 7)) (0.9.12)
Requirement already satisfied: elasticsearch~=8.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 8)) (8.1.3)
Requirement already satisfied: XlsxWriter~=1.3.6 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 9)) (1.3.9)
Requirement already satisfied: marshmallow~=3.13.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 10)) (3.13.0)
Requirement already satisfied: marshmallow-dataclass[union]~=8.5.6 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 11)) (8.5.8)
Requirement already satisfied: pyflakes==2.2.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 14)) (2.2.0)
Requirement already satisfied: flake8==3.8.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 15)) (3.8.1)
Requirement already satisfied: pep8-naming==0.7.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 16)) (0.7.0)
Requirement already satisfied: pytest>=3.6 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 17)) (7.1.2)
Requirement already satisfied: jsonschema==3.2.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 18)) (3.2.0)
Requirement already satisfied: marshmallow-jsonschema~=0.12.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 19)) (0.12.0)
Requirement already satisfied: marshmallow-union~=0.1.15 in ./env/detection-rules-build/lib/python3.8/site-packages (from -r requirements.txt (line 20)) (0.1.15.post1)
Requirement already satisfied: lark-parser~=0.11.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from eql==0.9.12->-r requirements.txt (line 7)) (0.11.3)
Requirement already satisfied: mccabe<0.7.0,>=0.6.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from flake8==3.8.1->-r requirements.txt (line 15)) (0.6.1)
Requirement already satisfied: pycodestyle<2.7.0,>=2.6.0a1 in ./env/detection-rules-build/lib/python3.8/site-packages (from flake8==3.8.1->-r requirements.txt (line 15)) (2.6.0)
Requirement already satisfied: flake8-polyfill<2,>=1.0.2 in ./env/detection-rules-build/lib/python3.8/site-packages (from pep8-naming==0.7.0->-r requirements.txt (line 16)) (1.0.2)
Requirement already satisfied: attrs>=17.4.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (21.4.0)
Requirement already satisfied: pyrsistent>=0.14.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (0.18.1)
Requirement already satisfied: setuptools in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (62.1.0)
Requirement already satisfied: six>=1.11.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from jsonschema==3.2.0->-r requirements.txt (line 18)) (1.16.0)
Requirement already satisfied: charset-normalizer~=2.0.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from requests~=2.27->-r requirements.txt (line 4)) (2.0.12)
Requirement already satisfied: idna<4,>=2.5 in ./env/detection-rules-build/lib/python3.8/site-packages (from requests~=2.27->-r requirements.txt (line 4)) (3.3)
Requirement already satisfied: certifi>=2017.4.17 in ./env/detection-rules-build/lib/python3.8/site-packages (from requests~=2.27->-r requirements.txt (line 4)) (2021.10.8)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from requests~=2.27->-r requirements.txt (line 4)) (1.26.9)
Requirement already satisfied: elastic-transport<9,>=8 in ./env/detection-rules-build/lib/python3.8/site-packages (from elasticsearch~=8.1->-r requirements.txt (line 8)) (8.1.2)
Requirement already satisfied: typing-inspect>=0.7.1 in ./env/detection-rules-build/lib/python3.8/site-packages (from marshmallow-dataclass[union]~=8.5.6->-r requirements.txt (line 11)) (0.7.1)
Requirement already satisfied: typeguard in ./env/detection-rules-build/lib/python3.8/site-packages (from marshmallow-dataclass[union]~=8.5.6->-r requirements.txt (line 11)) (2.13.3)
Requirement already satisfied: pluggy<2.0,>=0.12 in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (1.0.0)
Requirement already satisfied: py>=1.8.2 in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (1.11.0)
Requirement already satisfied: tomli>=1.0.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (2.0.1)
Requirement already satisfied: packaging in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (21.3)
Requirement already satisfied: iniconfig in ./env/detection-rules-build/lib/python3.8/site-packages (from pytest>=3.6->-r requirements.txt (line 17)) (1.1.1)
Requirement already satisfied: typing-extensions>=3.7.4 in ./env/detection-rules-build/lib/python3.8/site-packages (from typing-inspect>=0.7.1->marshmallow-dataclass[union]~=8.5.6->-r requirements.txt (line 11)) (4.2.0)
Requirement already satisfied: mypy-extensions>=0.3.0 in ./env/detection-rules-build/lib/python3.8/site-packages (from typing-inspect>=0.7.1->marshmallow-dataclass[union]~=8.5.6->-r requirements.txt (line 11)) (0.4.3)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in ./env/detection-rules-build/lib/python3.8/site-packages (from packaging->pytest>=3.6->-r requirements.txt (line 17)) (3.0.8)
RELEASE:
./env/detection-rules-build/bin/python -m detection_rules dev build-release --generate-navigator

█▀▀▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄   ▄      █▀▀▄ ▄  ▄ ▄   ▄▄▄ ▄▄▄
█  █ █▄▄  █  █▄▄ █    █   █  █ █ █▀▄ █      █▄▄▀ █  █ █   █▄▄ █▄▄
█▄▄▀ █▄▄  █  █▄▄ █▄▄  █  ▄█▄ █▄█ █ ▀▄█      █ ▀▄ █▄▄█ █▄▄ █▄▄ ▄▄█

[+] Building package 8.3
 - 12 rules excluded from package
Rule changes detected!
 - 17 changed rules
 - 4 new rules
 - 0 newly deprecated rules
run `build-release --update-version-lock` to update version.lock.json and deprecated_rules.json
Package saved to: /Users/stryker/workspace/Elastic/detection-rules/releases/8.3
- sha256: 15aaf0fd89754cd0497fa30d35d4954a3c64d6d94d354beae10cedaa06ec9686
- 679 rules included
rm -rf dist
mkdir dist
cp -r releases/*/*.zip dist/
(detection_dev) ➜  detection-rules git:(1765-fr-move-to-setuppy-installation)
```

</details>
